### ISSUE TITLE
please Add Previous button feature (1 step back)

### ISSUE THREAD

**Issue by KrishnaKesharwani** [xbsoftware/enjoyhint#131]
Hello Guys 
Overall Enjoy Hint Awesome...

But 1 are missing like previous button...
please add previous button in action (1 Step back)
how can add one more button on action.

**Comment by sabatale** [comment]
Did you manage to find a solution? @KrishnaKesharwani

**Comment by sabatale** [comment]
@KrishnaKesharwani You can find this feature here: [https://github.com/darron1217/enjoyhint.js](https://github.com/darron1217/enjoyhint.js)

**Comment by sdraper69** [comment]
Yes make this official! Im not sure if this is still being developed but within 2 minutes of playing it seems like an obvious ommision

**Comment by tsemashkoXB** [comment]
Hi @KrishnaKesharwani , thanks for reporting this and sorry for the long silence.

A **Previous** button is available and can be controlled per step with `showPrev` / `prevButton`.
